pkg/newstorage/store.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package storage
```

Comment:
You should name the parent folder storage. To make this work, you need a grandparent folder with a different name.

Reply:
Updated the package name to match the folder name.